lib/runners/processor.rb

```diff
@@ -226,6 +226,15 @@ def add_warning_if_deprecated_options(keys, doc:)
       end
     end
 
+    def add_warning_for_deprecated_linter(deadline: nil, alternative: nil)
+      deadline_str = deadline ? deadline.strftime("on %B %-d, %Y") : "in the near future"
+      add_warning <<~MSG.strip, file: ci_config_path_name
+        DEPRECATION WARNING!!!
+        The support for #{analyzer_name} is deprecated. Sider will drop these versions #{deadline_str}.
+        Please consider to use an alternative tool #{alternative}.
```

Comment:
[suggestion] What about fixing grammar like this?

```suggestion
        Please consider using an alternative tool #{alternative}.
```

Reply:
Thanks 💦